### DISCUSSION TITLE
Montage view & playback

### DISCUSSION THREAD

**Discussion by ronisaacson** [blakeblackshear/frigate#6442] [Ideas]
I recently switched all of my sites from a different NVR (another popular software package which I won't name) to Frigate and I couldn't be happier! There's one key feature that I miss though and it's a "montage" view, for both live & playback.

The montage view was mentioned in #6325, where you just want to see a live view of all of your cameras in a fixed layout. If I'm remote and want to peek in on what's going on, this is the most useful view. It also mimics what you'd get from a monitoring station where someone is keeping an eye on a whole space.

Unlike birdseye, the fixed layout is important because the automatic switching can make it very confusing to know what you're looking at, and you're at risk of missing undetected motion. Ideally this would be a grid layout that can responsively compress and an outline or highlight to indicate any zones with active events.

The other montage use case is for playback. Imagine the same view as above where you can scroll in the timeline and review the recordings from one or more cameras in sync with each other. This is useful when reviewing activity that might span multiple cameras, or when you're looking for something and you're not sure which camera might have picked it up.

**Comment by NickM-27** [comment]
> Unlike birdseye, the fixed layout is important because the automatic switching can make it very confusing to know what you're looking at, and you're at risk of missing undetected motion. Ideally this would be a grid layout that can responsively compress and an outline or highlight to indicate any zones with active events.

you can of course configure birdseye to be continuous as opposed to automatically show and hide cameras based on activity

**Reply by ronisaacson** [reply]
Yeah but it's still a single composed image, so you can't click on a camera view to zoom to that camera, and the layout isn't responsive based on screen size. A fixed birdseye view isn't really comparable to a grid-style montage of live views.

**Reply by NickM-27** [reply]
Sure, I was just making it clear that birdseye does support that feature.
